deploy/playbooks/examples/deploy_vagrant.yml

```diff
@@ -1,33 +1,17 @@
 ---
-
-# install python2.7 on xenial nodes
```

Comment:
Ah, I ran the playbook on a Xenial VPS in the Sepia lab.  python2.7 gets installed during the cloud-init/downburst dance which is why it worked.  I'll put it back.